### DISCUSSION TITLE
Welcome to SwitchHosts Discussions!

### DISCUSSION THREAD

**Discussion by oldj** [oldj/SwitchHosts#522] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meltanian** [comment]
Hi, i find an app in MicrosoftStore 
Micro SwitchHosts - it looks like your app but with other name and with price
If it is not you - it is a stealing

**Reply by oldj** [reply]
That was not uploaded by me, thanks for letting me know...

**Comment by meltanian** [comment]
You are welcome! And thanks for coll soft)Надіслано з пристрою Galaxy
-------- Оригінальне повідомлення --------Від: oldj ***@***.***> Дата: 14.04.23  13:02  (GMT+02:00) Кому: oldj/SwitchHosts ***@***.***> Копія: Vadim Dubovitskiy ***@***.***>, Comment ***@***.***> Тема: Re: [oldj/SwitchHosts] Welcome to SwitchHosts Discussions! (Discussion #522) 
That was not uploaded by me, thanks for letting me know...

**Comment by ghost** [comment]
Hey there!
